src/Settings/Aviant.SettingsGroup.json

```diff
@@ -1,84 +1,95 @@
-[
-{
-    "name":             "showTrafficIndicators",
-    "shortDesc": "Show traffic indicators",
-    "longDesc":  "Shows traffic indicators, including conflict radius and line guides",
-    "type":             "bool",
-    "default":     true
-},
-{
-    "name":             "showWinchControlMenu",
-    "shortDesc": "Show winch control menu",
-    "longDesc":  "Shows the winch control menu buttons and slider",
-    "type":             "bool",
-    "default":     true
-},
-{
-    "name":             "horizontalConflictDistance",
-    "shortDesc": "Horizontal conflict distance",
-    "longDesc":  "The horizontal distance to nearby traffic inside which the situation is considered a conflict, and traffic indicators turn red. Twice this distance is used for proximity detection, meaning that the indicators will turn yellow. ",
-    "type":             "uint32",
-    "min":              0,
-    "units":            "m",
-    "default":          4000
-},
-{
-    "name":             "verticalConflictDistance",
-    "shortDesc": "Horizontal conflict distance",
-    "longDesc":  "The vertical distance to nearby traffic inside which the situation is considered a conflict, and traffic indicators turn red. Twice this distance is used for proximity detection, meaning that the indicators will turn yellow. ",
-    "type":             "uint32",
-    "min":              0,
-    "units":            "m",
-    "default":          600
-},
-{
-    "name":         "rssiWarning",
-    "shortDesc":    "Local (drone) RSSI below this value will trigger a warning",
-    "type":         "double",
-    "default":      -90,
-    "units":        "dBm",
-    "min":          -128.0,
-    "max":          0.0
-},
-{
-    "name":         "rssiAlert",
-    "shortDesc":    "Local (drone) RSSI below this value will trigger an alert",
-    "type":         "double",
-    "default":      -105,
-    "units":        "dBm",
-    "min":          -128.0,
-    "max":          0.0
-},
-{
-    "name":                "missionToolsInsecureHttps",
-    "shortDesc":           "Insecure HTTPS connections",
-    "longDesc":            "Do not validate server certificate for mission tools HTTPS connection",
-    "type":                "bool",
-    "default":             false,
-    "qgcRebootRequired":   true
-},
-{
-    "name":                "missionToolsUrl",
-    "shortDesc":           "URL of mission tools",
-    "type":                "string",
-    "default":             ""
-},
-{
-    "name":                "missionToolsToken",
-    "shortDesc":           "Authentication token for mission tools",
-    "type":                "string",
-    "default":             ""
-},
-{
-    "name":                "hideBreachReturnPoint",
-    "shortDesc":           "Hide breach return point",
-    "longDesc":            "Hide breach return point that is not used by PX4 (use rally points)",
-    "type":                "bool",
-    "default":             true
-}
-]
-}
+    "version": 1,
```

Comment:
Split indentation to separate commit so I can see what's changed

Reply:
or dont indent

Reply:
Reverted the indent, didnt notice that it was autoformated on save